### ISSUE TITLE
Filemoon resolver not working

### ISSUE THREAD

**Issue by vargalex** [Gujal00/ResolveURL#628]
The filemoon resolver resolves the URL from the "packed" javascript funcion (with jsunpack). But this URL returns with HTTP 404. In the browser the final URL is different, but I have not found what changes the "t" and "e" URL parameters. Can you check it please? One example for filemoon link: `https://filmpapa.filmadatbazis.site/vegzetes-arulas/`

**Comment by Kliker12345** [comment]
Everything is fine Filemoon working.. you just need to play video and click download button on player and you will see filemoon link

There is for yours video: 

you can use like this: https://filemoon.sx/download/b0vz15o9xueg
or change "download" to "d"
https://filemoon.sx/d/b0vz15o9xueg

both working

**Comment by vargalex** [comment]
@Kliker12345 I don't understand your answer. The filemoon resolver [resolves the URL with e](https://github.com/Gujal00/ResolveURL/blob/master/script.module.resolveurl/lib/resolveurl/plugins/filemoon.py#L45). I want to use the resolveURL from Kodi, not playing from a browser with the embedded jwPlayer... When I call the resolveURL with the original URL: [https://filemoon.sx/e/b0vz15o9xueg](https://filemoon.sx/e/b0vz15o9xueg), than cannot resolve it, because the resolved URL returning with HTTP 404.

**Comment by Kliker12345** [comment]
Ok, I'm sorry. I thought you were using the url from the site and not directly from filemoon. I just tested and not working.

**Comment by vargalex** [comment]
@Kliker12345 Sorry, you are right. I have forgot, that this site requires registration. I this example this is the Filemoon url: https://filemoon.sx/e/b0vz15o9xueg

**Comment by Gujal00** [comment]
@vargalex please test with https://github.com/Gujal00/ResolveURL/commit/ed739e70b52cc66caea42fcf36e67959e2ab7fd8 and report back

**Comment by vargalex** [comment]
@Gujal00 Thanks for the fix! I have tested with multiple sources, all resolved perfectly! (In my addons for Hungarian sites are many filemoon streams. :) )
